ports/servoshell/Cargo.toml

```diff
@@ -35,11 +35,12 @@ OriginalFilename = "servo.exe"
 ProductName = "Servo"
 
 [features]
-default = ["max_log_level", "webgpu", "webxr"]
+default = ["libservo/clipboard", "js_jit", "max_log_level", "webgpu", "webxr", "libservo/vello_cpu"]
```

Comment:
We already enable vello_cpu down at libservo fetures, should we do the same for clipboard?

Reply:
Aha, but we should not need vello_cpu here because it's already specified down there unconditionally.